libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewInitializedObserver.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.navigation.dropin
+
+interface NavigationViewInitializedObserver {
```

Comment:
```suggestion
interface NavigationViewInitializedCallback {
```

Reply:
Actually, I'd be in favor of removing this callback for now, refs https://github.com/mapbox/mapbox-navigation-android/pull/5133/files#r754397691.